[EDU-3196] feat: add understand metrics idns (#499)
aziontech/docs: src/content/docs/en/pages/secure-journey/troubleshoot/intelligent-dns-understand-metrics.mdx

---

> title: Understand metrics

Reviewer: ```suggestion
title: Understand Intelligent DNS metrics
```

Reviewer: Como temos outras métricas, dá para especificar no h1

description: You can continuously monitor your applications through Azion Observe products.
meta_tags: observe, metrics, applications, secure, intelligent dns
namespace: docs_secure_understand_metrics_dns
permalink: /documentation/products/secure/troubleshoot/intelligent-dns-understand-metrics/
menu_namespace: secureMenu
---

import Button from '~/components/Button.astro'

> Once you host your domains and create zones on [Intelligent DNS](/en/documentation/products/intelligent-dns/), you can also count on [Observe](/en/documentation/products/observe/) products to monitor how WAF is processing requests made to your applications.

Reviewer 2: É host domain o termo correto? Eu pensei em algo mais geral tipo `create records`.

Author: é sim! é a especificade

> Once you host your domains and create zones on [Intelligent DNS](/en/documentation/products/intelligent-dns/), you can also count on [Observe](/en/documentation/products/observe/) products to monitor how WAF is processing requests made to your applications.

Reviewer: ```suggestion
Once you host your domains and create zones on [Intelligent DNS](/en/documentation/products/intelligent-dns/), you can also rely on [Observe](/en/documentation/products/observe/) products to monitor how WAF is processing requests made to your applications.
```

Reviewer: essa aqui é só gosto mesmo, pode desconsiderar se não concorda xD

- [Real-Time Metrics](/en/documentation/products/real-time-metrics/): access to charts with aggregated data to explore behaviors throughout longer periods of data storage.

> - [GraphQL API](/en/documentation/products/graphql-api-overview/): access to aggragated and raw data via API with customized queries with only the data you request.

Reviewer 3: ```suggestion
- [GraphQL API](/en/documentation/products/graphql-api-overview/): access to aggregated and raw data via API with customized queries with only the requested data.
```

## Intelligent DNS

To monitor queries received by your configured DNS:

> <Button href="/en/documentation/products/real-time-metrics/#intelligent-dns" text="view aggregated metrics" variant="secondary"></Button> <br></br>

Reviewer: ```suggestion
<Button href="/en/documentation/products/real-time-metrics/#intelligent-dns" text="go to aggregated metrics reference" variant="secondary"></Button> <br></br>
```

> <Button href="/en/documentation/products/graphql-api-features/#datasets" text="query data" variant="secondary"></Button> <br></br>

Reviewer: ```suggestion
<Button href="/en/documentation/products/graphql-api-features/#datasets" text="go to query data reference" variant="secondary"></Button> <br></br>
```
